pyproject.toml

```diff
@@ -119,10 +121,15 @@ wheels = [
     "tox>=4.26.0",
 ]
 
+# FIXME Temporarily disable cp313t/cp314t wheels due to instability concerns.
+ # See matching FIXMEs in azure-pipelines.yml and setup_build.py.
 [tool.cibuildwheel]
 skip = [
-    "cp314t-*",
     "cp310-win_arm64", #Skipped due to unavailability of Python binary and NumPy for Win-ARM64
+    "cp314t-*",  # Comment out to build cp314t wheels
+]
+enable = [
+    # "cpython-freethreading",  # Uncomment to build cp313t wheels
```

Comment:
I'd rather not add commented code for now. I'm well aware of this option any way, I'm confident we won't forget about it :)